MoltenVK/MoltenVK/GPUObjects/MVKRenderPass.mm

```diff
@@ -1031,28 +1148,34 @@
 	}
 
 	// Link attachments to subpasses
-	for (auto& att : _attachments) {
-		att.linkToSubpasses();
-	}
+	linkAttachments();
 }
 
 MVKRenderPass::MVKRenderPass(MVKDevice* device, const VkRenderingInfo* pRenderingInfo) : MVKVulkanAPIDeviceObject(device) {
 
 	_renderingFlags = pRenderingInfo->flags;
 
 	// Add attachments first so subpasses can access them during creation
+	// Attachments alternate between color and resolve, even if resolved are unused, then with depth, depth resolve, stencil, and stencil resolve at the end.
```

Comment:
This is probably where the comment I mentioned above referring to `MVKFramebuffer` and `MVKCommandEncoder` should go.

Reply:
Thanks. Fixed.